contrib/cirrus/setup_environment.sh

```diff
@@ -141,6 +141,41 @@ case "$TEST_ENVIRON" in
             warn "Forcing CGROUP_MANAGER=cgroupfs"
             echo "CGROUP_MANAGER=cgroupfs" >> /etc/ci_environment
         fi
+        # TODO: For the foreseeable future, need to support running tests
+        # with and without the latest netavark/aardvark.  Once they're more
+        # stable and widely supported in Fedora, they can be pre-installed
+        # from its RPM at VM image build-time.
+        if [[ "$TEST_ENVIRON" =~ netavark ]]; then
+            for info in "netavark $NETAVARK_BRANCH $NETAVARK_URL $NETAVARK_DEBUG" \
+                        "aardvark-dns $AARDVARK_BRANCH $AARDVARK_URL $AARDVARK_DEBUG"; do
+
+                read _name _branch _url _debug <<<"$info"
+                req_env_vars _name _branch _url _debug
+                msg "Downloading latest $_name from upstream branch '$_branch'"
+                # Use identifiable archive filename in of a get_ci_env.sh environment
+                curl --fail --location -o /tmp/$_name.zip "$_url"
+
+                # Needs to be in a specific location
+                # ref: https://github.com/containers/common/blob/main/pkg/config/config_linux.go#L39
+                _pdir=/usr/local/libexec/podman
+                mkdir -p $_pdir
+                cd $_pdir
+                msg "$PWD"
+                unzip /tmp/$_name.zip
+                if ((_debug)); then
+                    warn "Using debug $_name binary"
+                    mv $_name.debug $_name
+                else
+                    rm $_name.debug
+                fi
+                chmod 0755 $_pdir/$_name
+                cd -
+            done
+
+            restorecon -F -v $_nvdir
+            msg "Forcing NETWORK_BACKEND=netavark in all subsequent environments."
+            echo "NETWORK_BACKEND=netavark" >> /etc/ci_environment
```

Comment:
Can we be a little more careful with envariables? Like, prepend them with `PODMAN_` or `CIRRUS_` or something sane? `NETWORK_BACKEND` could conceivably be used by anyone anywhere, and maybe next week `ping` or `netcat` or even `ls` could start using it and push out a version that works completely differently when set. We should be polite about namespaces, even if other tools (\*cough\* systemd) are not.

Reply:
I don't disagree, however this name was decided well in advance of this PR.  @baude is there anything we can do about this at this point?